LiteCore/Query/N1QL_Parser/n1ql.leg

```diff
@@ -345,6 +345,11 @@ propertyPath =
 propertyName
     = IDENTIFIER
 
+metaPath =
```

Comment:
Can propertyPath be used here?

Reply:
No. It doesn't work

Reply:
propertyPath does some weird quoting and stuff that doesn't work with the way META expression path is parsed. It breaks most META expression tests